README.md

```diff
@@ -254,6 +256,14 @@ Job config for test jobs run for each PR in prow can be found [here](https://git
   kubectl logs csi-secrets-store-secrets-store-csi-driver-7x44t secrets-store
   ```
 
+## Communication
```

Comment:
How about we add this to the top of the readme before `#Features`, something like this?

```
## Want to help?
Join us to help define the direction and implementation of this project!
- Join the [`#csi-secrets-store`]((https://kubernetes.slack.com/messages/csi-secrets-store)
  channel on [Kubernetes Slack](https://kubernetes.slack.com/).
- Use [GitHub Issues](https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues)
  to file bugs, request features, or ask questions asynchronously.
```

Reply:
updated!